src/GafferScene/Instancer.cpp

```diff
@@ -1286,3 +1445,11 @@ Instancer::PrototypeScope::PrototypeScope( const Gaffer::ObjectPlug *enginePlug,
 		set( ScenePlug::scenePathContextName, prototypeRoot );
 	}
 }
+
+void Instancer::plugDirtied( const Gaffer::Plug *plug )
+{
+	if( plug->parent() == outPlug() )
```

Comment:
Remind me to talk to you about this later - I think we can do better than invalidate the capsule every time the out plug changes. It should be sufficient to depend only on the poor man's hash of `prototypesPlug()` and the exact hash of the points object. The goal is not to invalidate the capsule for any old change in the main input scene.